client/src/components/footer/footer.js

```diff
@@ -224,10 +229,11 @@ class Footer extends Component {
                         : "block",
                   }}
                 >
-                  Next -{" "}
+                  Next
```

Comment:
Based on the Figma, it looks like the button doesn't have any background color. 
![Screenshot 2024-10-03 at 11 50 09 AM](https://github.com/user-attachments/assets/86ed2b11-8f8e-450b-96a5-d34bf5ffa4b1)
![Screenshot 2024-10-03 at 11 50 16 AM](https://github.com/user-attachments/assets/0f8603fb-40bf-4cac-9da0-755472738cb4)

I'm seeing a gray background on these buttons however. I see you have a `tw-bg-bgwhite`, so that probably needs to be fixed
![Screenshot 2024-10-03 at 11 50 29 AM](https://github.com/user-attachments/assets/230a722d-785a-40e5-bc43-f45b06d5a8bc)
![Screenshot 2024-10-03 at 11 50 32 AM](https://github.com/user-attachments/assets/42918d2c-1b05-4857-b77d-f554cba90619)

Reply:
Also the angle of the color border looks a little different from the Figma - instead of a horizontal slant at the end, it should be flat